app/src/main/res/layout/activity_auth.xml

```diff
@@ -0,0 +1,233 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.05" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guidelineEnd"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.95" />
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/flowCentralText"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:constraint_referenced_ids="textLabelLetGet, textLabelFillOut"
+        app:flow_verticalGap="@dimen/margin_smallest"
+        app:flow_verticalStyle="spread_inside"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@id/guidelineEnd"
+        app:layout_constraintStart_toStartOf="@id/guidelineStart"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.22" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/textLabelLetGet"
+        style="@style/TextAppearance.AppCompat.White24"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/letsGetAcquaintedLabel"
+        app:layout_constraintBottom_toTopOf="@id/textLabelFillOut"
+        app:layout_constraintEnd_toEndOf="@id/guidelineStart"
+        app:layout_constraintStart_toStartOf="@id/guidelineEnd"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/textLabelFillOut"
+        style="@style/TextAppearance.AppCompat.White12"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/fillOutProfileInstruction"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@id/guidelineEnd"
+        app:layout_constraintStart_toStartOf="@id/guidelineStart"
+        app:layout_constraintTop_toBottomOf="@id/textLabelLetGet" />
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/flowEmailPassRemember"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:constraint_referenced_ids="emailInputLayout,passInputLayout, checkBox"
+        app:flow_verticalStyle="spread_inside"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@id/guidelineEnd"
+        app:layout_constraintStart_toStartOf="@id/guidelineStart"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.42" />
+
+    <com.google.android.material.textfield.TextInputLayout
```

Comment:
Cursor color does not meet figma design.
<img width="439" alt="image" src="https://github.com/user-attachments/assets/57b71b98-3428-4b2b-9bdf-468b13f843e5">

Reply:
![1](https://github.com/user-attachments/assets/dd412df5-a5cc-42a9-835e-3c9a5a8ec80b)
Sorry. I have white cursore.